lib/msal-node/src/config/ClientConfiguration.ts

```diff
@@ -3,15 +3,17 @@
  * Licensed under the MIT License.
  */
 import {
-    AuthOptions,
     LoggerOptions,
     INetworkModule,
     LogLevel,
 } from '@azure/msal-common';
 import { NetworkUtils } from '../utils/NetworkUtils';
 import { CACHE } from '../utils/Constants';
 
-export type NodeAuthOptions = AuthOptions;
```

Comment:
Shouldn't need this with new config changes in #1575

Reply:
yep reverted this change.